sorcery.gemspec

```diff
@@ -37,6 +37,8 @@ Gem::Specification.new do |s|
   s.add_dependency 'bcrypt', '~> 3.1'
   s.add_dependency 'oauth', '~> 0.4', '>= 0.4.4'
   s.add_dependency 'oauth2', '~> 1.0', '>= 0.8.0'
+  s.add_dependency 'bcrypt', '~> 3.1'
```

Comment:
@ebihara99999 Duplicate `bcrypt` dependency, likely from a merge. I would double check anything that changed on your merge commits.

Reply:
@athix I'm sorry to add the wrong dependency. I've fixed it just now